src/object.d

```diff
@@ -529,6 +529,68 @@ bool __equals(T1, T2)(T1[] lhs, T2[] rhs)
     assert(a1 != a2);
 }
 
+void __move_post_blt(S)(ref S newLocation, ref S oldLocation) nothrow
```

Comment:
Can this be private and have the compiler bypass the protection?

Reply:
It can't be private because `std.algorithm.mutation.move` has to be able to call it:

> The `move` family of functions defined in `std.algorithm` MUST be updated to add a call to `__move_post_blt`.